controllers/bpfman-agent/common.go

```diff
@@ -228,11 +231,37 @@ func getClientset() (*kubernetes.Clientset, error) {
 	return clientset, nil
 }
 
-func getInterfaces(interfaceSelector *bpfmaniov1alpha1.InterfaceSelector, ourNode *v1.Node) ([]string, error) {
+func interfaceInExcludeList(interfaceSelector *bpfmaniov1alpha1.InterfaceSelector, intf string) bool {
+	for _, i := range interfaceSelector.InterfacesDiscoveryConfig.ExcludeInterfaces {
+		if i == intf {
+			return true
+		}
+	}
+	return false
+}
+
+func getInterfaces(interfaceSelector *bpfmaniov1alpha1.InterfaceSelector, ourNode *v1.Node, discoveredInterfaces *sync.Map) ([]string, error) {
```

Comment:
I think this is okay for now, but when you support attaching bpf programs inside network namespaces, I think getInterfaces will need to return both the interface name and the network namespace path.